ledger/acctdeltas_test.go

```diff
@@ -59,7 +59,7 @@ func checkAccounts(t *testing.T, tx trackerdb.TransactionScope, rnd basics.Round
 	require.NoError(t, err)
 	require.Equal(t, r, rnd)
 
-	aor, err := tx.Testing().MakeAccountsOptimizedReader()
+	aor, err := tx.MakeAccountsOptimizedReader()
```

Comment:
Why is this removed here and not elsewhere?
Maybe remove the Testing() from the interface as well? It is not a good idea to have something for testing in the production code.

Reply:
This comes from early refactoring too. 
The methods used only in testing are behind a testing interface. 
`MakeAccountsOptimizedReader` was promoted and removed from the testing only.